config/kernel-misc-minor.m4

```diff
@@ -0,0 +1,26 @@
+dnl #
+dnl # Determine an available miscellaneous minor number which can be used
+dnl # for the /dev/zfs device.  This is needed because kernel module
+dnl # auto-loading depends on registering a reserved non-conflicting minor
+dnl # number.  Start with a large known available unreserved minor and work
+dnl # our way down to lower value if a collision is detected.
+dnl #
+AC_DEFUN([ZFS_AC_KERNEL_MISC_MINOR], [
+	AC_MSG_CHECKING([for available /dev/zfs minor])
+
+	for i in $(seq 249 -1 200); do
+		if ! grep -q "#define.*_MINOR.*$i" \
```

Comment:
`"^#define\s\+.*_MINOR\s\+.*$i"` would be a bit more exact and prevent potential future false negatives (they currently match the same device minor numbers).